.gitignore

```diff
@@ -396,3 +396,5 @@ model.index\+
 # local environment settings used by e.g. Visual Studio Code
 /.env
 /doc/source/local_config.json
+/pyaedtenv/**
```

Comment:
@ramin4667 I would not add these 2 lines. In a different pull request we can consider it, but I would keep it clean and focus in the new implementation